src/inference/dev_api/openvino/runtime/internal_properties.hpp

```diff
@@ -29,6 +29,13 @@ static constexpr Property<std::vector<PropertyName>, PropertyMutability::RO> sup
  */
 static constexpr Property<std::vector<PropertyName>, PropertyMutability::RO> caching_properties{"CACHING_PROPERTIES"};
 
+/**
+ * @brief Read-only property to get a std::vector<PropertyName> of properties
+ * which should affect the loading time from cache
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<std::vector<PropertyName>, PropertyMutability::RO> caching_with_mmap{"CACHING_WITH_MMAP"};
```

Comment:
why is it a list of property names? I suppose it can be just bool ?

Reply:
Done